mysql-test/std_data/keyring_vault_confs/keyring_vault.conf

```diff
@@ -0,0 +1,4 @@
+vault_url = https://127.0.0.1:8200
```

Comment:
Yes, by not running in parallel I meant all kinds of parallel execution - separate machines, parallel runs on a single machine. This would be very difficult to have some kind of port dispatcher that would assign unique Vault server to each keyring_vault being run. The dispatcher would need to be placed on a separate server and mysql-test-run would need to be extended so to be able to communicate with such a server. 
That would be very difficult. We had similar problem in Oracle. I guess this is the simplest solution. Allowing running in parallel would require some central server with a port pool